packages/api-client/src/account/AccountAPI.ts

```diff
@@ -125,7 +125,7 @@ export class AccountAPI {
    * Retrieve all TURN server addresses and credentials.
    * Clients are expected to do a DNS lookup to resolve the IP addresses of the given hostnames
    *
-   * @param limit Limits the number of ICE-Candidates returned. [1..10]
```

Comment:
Please remove the dashes in TS files, see here why: https://github.com/wireapp/wire-webapp/pull/7904#discussion_r358145110

Reply:
They were added automatically, do you know which rule is that?